app/src/controllers/storage.js

```diff
@@ -0,0 +1,29 @@
+const errorToProblem = require('../components/errorToProblem');
+const { storageService } = require('../services');
+
+const SERVICE = 'StorageService';
+
+/**
+ * The Storage Controller
+ */
+const controller = {
+  async getEncryption(req, res, next) {
+    try {
+      const response = await storageService.getBucketEncryption();
+      res.status(200).json(response);
+    } catch (e) {
+      next(errorToProblem(SERVICE, e));
+    }
+  },
+
+  async putEncryption(req, res, next) {
+    try {
+      const response = await storageService.putBucketEncryption();
+      res.status(200).json(response);
+    } catch (e) {
+      next(errorToProblem(SERVICE, e));
+    }
+  },
+};
```

Comment:
It's good to keep this code around, but probably scope it out to a different branch of work for now as we still need to get a better sense of where we're headed with encryption processes in general.